stdlib/InteractiveUtils/src/InteractiveUtils.jl

```diff
@@ -15,7 +15,6 @@ using Base: unwrap_unionall, rewrap_unionall, isdeprecated, Bottom, show_unquote
 
 using Markdown
 using LinearAlgebra  # for peakflops
-import Pkg, OldPkg
```

Comment:
Should update the project file for this package (remove `Pkg`)

Reply:
Actually we should rewrite the code to use `Pkg` instead of `OldPkg`. What would be an appropriate status-like function to call?

Reply:
But `Pkg` depends on `REPL` which depends on `InteractiveUtils`. I thought we wanted to not have a cycle.

Reply:
It works as long as `InteractiveUtils` doesn't try to import Pkg at load time. Or, other options would be to try to break the REPL->InteractiveUtils dependency, or just not have `versioninfo` show package versions.

Reply:
>  or just not have versioninfo show package versions.

+1